internal/app/machined/pkg/controllers/siderolink/manager.go

```diff
@@ -198,10 +206,22 @@ func (ctrl *ManagerController) Run(ctx context.Context, r controller.Runtime, lo
 				}
 			}()
 
+			uniqTokenRes, rdrErr := safe.ReaderGetByID[*runtime.UniqueMachineToken](ctx, r, runtime.UniqueMachineTokenID)
+			if rdrErr != nil && !state.IsNotFoundError(rdrErr) {
+				return nil, fmt.Errorf("failed to get unique token: %w", rdrErr)
```

Comment:
we should skip here if the resource is not found, i.e. controller should only start when the resource is present?

Reply:
It's a leftover from the previous iteration. Thanks for noticing, I also removed the code below which worked with assumption that uniqueToken resource may not be present.